t/t6025-merge-symlinks.sh

```diff
@@ -17,14 +16,14 @@ test_expect_success 'setup' '
     git commit -m initial &&
     git branch b-symlink &&
     git branch b-file &&
-    l="$(printf file | 
-    git hash-object -t blob -w --stdin)" &&
+    l="$(printf >file 
```

Comment:
This is incorrect in two ways: `printf file` outputs `file` without line feed, but `printf` on its own most likely errors out. You have to redirect _in addition_ to keeping the original argument. Also, since you want this statement to be run on its own, not in a pipe, it needs to be on its own line:

```sh
printf file >out &&
l="$(git hash-object -t blob -w --stdin <out)" &&
```